### ISSUE TITLE
Colour Board Controls Fail on Some Machines

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#1513]
Claudio Giovenzana reports:

> unfortunately with the new beta the  "Touch Bar Widget:Color Board Panel Toggle Button & 4x Puck controlâ€ still went crazy when sliding on the touchbar buttons for color adjustments..
> Iâ€™ll send you this message on a email too with a Gif attached

![sep-06-2018 09-26-26](https://user-images.githubusercontent.com/22286696/45192912-a9487b80-b28e-11e8-9a0d-d7a9430333e0.gif)

**Console Output:**
```
CommandPost v1.0.0-beta.33

Date Built:                     Sep  6 2018, 13:51:43
macOS Version:                  10.13.6
CommandPost Locale:             en
Current Keyboard Layout:        U.S.
Final Cut Pro Path:             /Applications/Final Cut Pro.app
Final Cut Pro Version:          10.4.3
Final Cut Pro Language:         en
Developer Mode:                 false

2018-09-06 09:15:37: 2018-09-06 09:15:36: -- Loading /Applications/CommandPost.app/Contents/Resources/extensions/cp/init.lua
2018-09-06 09:15:36: -- Loading extension: uielement
2018-09-06 09:15:37: 09:15:37         cp:     Loading Plugins...
2018-09-06 09:15:39: 09:15:39       scan:     ---------------------------------------------------------
2018-09-06 09:15:39:                          FINAL CUT PRO PLUGIN SCANNER:
2018-09-06 09:15:39:                          ---------------------------------------------------------
2018-09-06 09:15:39:                          * Scanning app-bundled plugins:
2018-09-06 09:15:40: 09:15:40                   * Loaded App Plugins from Cache: 0.125727
2018-09-06 09:15:40:                              * scanAppPlugins(en) took: 0.126093
2018-09-06 09:15:40:                           
2018-09-06 09:15:40:                          * Scanning system-installed plugins:
2018-09-06 09:15:40:                            * Loading System Motion Templates from Cache (135 items)
2018-09-06 09:15:40:                            * scanSystemMotionTemplates(cp.i18n.localeID: English [en]): 0.019309
2018-09-06 09:15:40:                            * Using Audio Units Cache (33 items)
2018-09-06 09:15:40:                            * scanSystemAudioUnits(cp.i18n.localeID: English [en]): 0.0046599999999994
2018-09-06 09:15:40:                              * scanSystemPlugins(cp.i18n.localeID: English [en]) took: 0.024759
2018-09-06 09:15:40:                           
2018-09-06 09:15:40:                          * Scanning user-installed plugins:
2018-09-06 09:15:40:                            * Using User Effects Presets Cache (45 items).
2018-09-06 09:15:40:                            * scanUserEffectsPresets(cp.i18n.localeID: English [en]): 0.0057119999999999
2018-09-06 09:15:42: 09:15:42                   * Loading User Motion Templates from Cache (3023 items)
2018-09-06 09:15:42:                            * scanUserMotionTemplates(cp.i18n.localeID: English [en]): 2.52595
2018-09-06 09:15:42:                              * scanUserPlugins(cp.i18n.localeID: English [en]) took: 2.532602
2018-09-06 09:15:42:                          ---------------------------------------------------------
2018-09-06 09:15:42: 09:15:42 ** Warning:   plugins: Slow Plugin: finalcutpro.setup.scan (2.706686)
2018-09-06 09:15:43: 09:15:43         cp:     Plugins Loaded.
2018-09-06 09:15:43: -- Done.
2018-09-06 09:17:27: -- Loading extension: chooser
2018-09-06 09:17:27: -- Loading extension: window
2018-09-06 09:18:52: ********
2018-09-06 09:18:52: 09:18:52 ERROR:colorInspe: Corrections Bar activation failed due to menu button timing out.
2018-09-06 09:18:52: ********
2018-09-06 09:19:01: ********
2018-09-06 09:19:01: 09:19:01 ERROR:colorInspe: Corrections Bar activation failed due to menu button timing out.
2018-09-06 09:19:01: ********
```

**Comment by latenitefilms** [comment]
FYI: I've just sent Claudio an email to see if he's still having issues, as I haven't been able to reproduce.

**Comment by latenitefilms** [comment]
Some users are reporting the exact same issue when using the Tangent or MIDI devices, so it's not specific to the Touch Bar - it's something funky in the FCPX API.

**Comment by randomeizer** [comment]
I think this is one of those cases when we have to go full Rx.

**Comment by latenitefilms** [comment]
I’m not sure if this is Rx-related, as it seems to work fine on some machines but not others. I have a feeling it might be related to region settings? Maybe some regions display the numbers differently than others?

**Comment by latenitefilms** [comment]
Fixed in #1747
